plugins/modules/cloud/misc/proxmox.py

```diff
@@ -68,28 +69,33 @@
   disk:
     description:
       - hard disk size in GB for instance
+      - If I(proxmox_default_behavior) is set to C(compatiblity) (the default value), this
+        option has a default of C(3).
```

Comment:
```suggestion
        option has a default of C(3). Note that the default value of I(proxmox_default_behavior) changes in
        community.general 2.0.0.
```
Better mention it too often than too little :)

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/850/commits/7a9d61e94ddd6d87b45b24b4679f0e6ba9e74880.